nestedtensor/nested/nested.py

```diff
@@ -282,7 +282,7 @@ def to_list(self):
     def to_tuple(self):
         return self._impl.to_tuple()
 
-    def to_tensor_mask(self, mask_dim=None):
+    def to_tensor_mask_tuple(self, mask_dim=None):
```

Comment:
Appending "_tuple" to the API prevents us from returning a list or NamedTuple in the future, i.e. graduate to returning a data structure that can behave similar to tuples but does more / things differently. It also seems overly verbose. What's you reasoning behind this? We also don't encode the return type into the torch.* APIs if I'm not mistaken.